src/components/blocks/MovieCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect }  from "react";
+import { Movie } from '../../assets/js/types';
+import { useDispatch } from "react-redux";
+import { getImage } from "../../assets/js/utils";
+
+import Loader from "./../Loader";
+import { getMovie, openFormForCreateOrUpdateOrDelete } from "../../store/reducers/movies";
+
+type Props = {
+    film: Movie
+}
+
+const MovieCard: React.FC<Props> = ({ film }) => {
+    const [img, setImg] = useState('')
+    const dispatch = useDispatch()
+    
+    useEffect(() => {
+        getImage(film.poster_path)
+            .then((url) => setImg(url))
+            .catch(() => setImg('https://via.placeholder.com/320x450/000000?text=Image+has+not+found'))
```

Comment:
i think that i have already have some similar logic. You can create some component to handle image with placeholder

Reply:
fixed it)